idd/Energy+.idd.in

```diff
@@ -69827,6 +69827,110 @@ SetpointManager:SingleZone:OneStageHeating,
        \required-field
        \type node
 
+SetpointManager:ReturnTemperature:ChilledWater,
+     \memo This setpoint manager is used to place a temperature setpoint on a
```

Comment:
This is a little repetitive and the first sentence says "fixed return . . ." and the second sentence says "which is scheduled."  How about something like (still a little clunky):
This setpoint manager is used to place a temperature setpoint on a plant supply outlet node based on a target return water setpoint. The setpoint manager attempts to achieve the desired return water temperature by adjusting the supply temperature setpoint based on the plant conditions at each system time step.